client/post-editor/editor-ground-control/history-button.jsx

```diff
@@ -6,32 +6,39 @@ import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
-import { flow } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import { openPostRevisionsDialog } from 'state/posts/revisions/actions';
+import { withAnalytics, recordTracksEvent } from 'state/analytics/actions';
+import EditorRevisionsDialog from 'post-editor/editor-revisions/dialog';
```

Comment:
`EditorRevisionsDialog` was refactored out of this component in https://github.com/Automattic/wp-calypso/pull/20823/files

Please make sure to sync this!

Reply:
I went ahead and pulled it out.